dags/test_slack_notifier.py

```diff
@@ -32,6 +38,8 @@ def task_fail():
     tags=["slack"],
     catchup=False,
 ) as dag:
+    dag.byline = "Critical DAG, must fix ASAP. Call Bob A. and Alice B. for help. Also, <@UMS32US1E>"
```

Comment:
answering my own question to say its someones member ID! i see theres an issue for future refinements, but could you add a quick comment here saying something like tag users by member id?

Reply:
I'm not sure if people will know to look here. I'll add something to the README though - does that sound ok?

Reply:
I was thinking here because when reading the line of code I didn't know it was tagging you without having seen it in the airflow channel.
if there is going to be readme updates in a future branch, it can go there then. its a cool feature!

Reply:
Ah, gotcha -- that isn't clear. I will add a comment about who that particular UID is, and I also added some docs here: https://github.com/cityofaustin/atd-airflow/pull/261/commits/2a10c45ff9d4a1e9fb689c490c4b8111215eb7dc.

Reply:
Thanks Chia -- how about this: https://github.com/cityofaustin/atd-airflow/pull/261/commits/d0aa04ae2db29953851c3e27ea35bfece4343186